### PR TITLE
Allow for newlines (\n) in /discord broadcast

### PR DESCRIPTION
**Pull request by ANutley**
Title is self explanatory.

### CODE REVIEW

**Comment by Vankka** [comment]
This is a breaking change to the way the broadcast command works, some users might have `\n` in the content of broadcast messages (while not wanting them to be converted to newlines)

**Review by Vankka — CHANGES_REQUESTED** [review]
See above

**Comment by Dinty1** [comment]
A config option set to false by default could solve that problem

**Review by Vankka — APPROVED** [review]
After deliberation, use cases where `\n` is wanted plain are unlikely enough that, having this minor breakage is fine.

File: src/main/java/github/scarsz/discordsrv/commands/CommandBroadcast.java [hunk 1]
```diff
@@ -45,7 +45,7 @@ public static void execute(CommandSender sender, String[] args) {
         TextChannel target = null;
 
         if (args.length > 0 && args[0].startsWith("#")) {
-            String raw = args[0].replace("#", "");
+            String raw = args[0].replace("#", "").replace("\\n", "\n");
```
Comment (Vankka) [inline src/main/java/github/scarsz/discordsrv/commands/CommandBroadcast.java:48]: This is only used to get the channel, thus it isn't neccasery to replace newlines here

Comment (ANutley) [reply src/main/java/github/scarsz/discordsrv/commands/CommandBroadcast.java]: Changed

**Pull request merged**